client/app/services/service-details/service-details-ansible.html

```diff
@@ -12,8 +12,8 @@ <h3 translate>Results</h3>
                   <div class="form-group">
                     <label class="control-label col-sm-3" translate>Status</label>
                     <div class="col-sm-8">
-                      <input class="form-control text-capitalize" readonly
-                             value="{{item.stack.status || ('Unknown' | translate)}}"/>
+                      <icon-status status="item.stack.status" success="['successful', 'succeeded', 'create_complete']"
```

Comment:
Presently not a long list, hence the deliberate inclusion for this first run. If it gets much longer tho you're spot on, const in the js is gonna be da way to play